content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+
+NOTE: a bit more text here about the demos in general.
+
+NOTE: still working on this list
+
+* link:https://github.com/kzantow[Keith Zantow] gave a live demo of
+  *link:/projects/blueocean[Blue Ocean] in Action*.
+* link:https://github.com/bitwiseman[Liam Newman] gave a live demo of
+  *Notifications with Jenkins Pipeline* based on his previous
+  link:/blog/2016/07/18/pipline-notifications/[blog post].
+* link:https://github.com/orrc[Christopher Orr] presented a lightning version of his
+  *Pipelines for building and deploying Android apps* talk
+  (link:link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-continuous_build_delivery_pip.pdf[Slides]).
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] showed a different way to
+  manage security with the *Ownership plugin for Jenkins*
+  (link:https://speakerdeck.com/onenashev/jw2016-ownership-plugin-demo[Slides])
+
+| 12:45 - 13:00
+| *Docker and Pipeline*
+| _Learn how to use Docker inside of Pipeline for clean, repeatable testing environments_
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| 13:00 - 13:15
+| *Git plugin - large repos, submodule authentication and more*
+| _Techniques for managing large Git repositories, Submodule authentication, Pipelines and Git_
+| link:https://github.com/markewaite[Mark Waite]
+
+| 13:15 - 13:30
+| *Freestyle to Pipeline*
+| _Overview of how easy it is to migrate from a confusing series of Freestyle Jobs to Jenkins Pipeline_
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| 13:30 - 13:45
+| *`package.json` and Jenkins*
+| _Using `package.json` to control your build; running tests, coverage and generating documentation in Jenkins_
+| link:https://github.com/cvega[Casey Vega]
+
+| 13:45 - 14:00
+| *Extending Pipeline with Libraries*
+| _When you have many jobs using similar configuration, it is natural to factor out the common parts into libraries. See some ways Pipeline lets you do this._
+| link:https://github.com/jglick[Jesse Glick]
+
+|===
+
+
+=== Thursday, September 15th
+[options=header]
+|===
+| Time | Session | Details | Presenter
+| 12:15 - 12:30
+| *A simpler way to define Jenkins Pipelines*
+| _Get to know a new way to define your Pipelines in a more configuration-like way!_
+| link:https://github.com/abayer[Andrew Bayer]
+
+| 12:30 - 12:45
+| *Multibranch Pipelines + Git symbolic-ref*
+| _Pipeline Multibranch Plugin is amazing, but is even better when used with
+Git symbolic references.
+| link:https://github.com/jh86[Jon Hermansen]
+
+| 12:45 - 13:00
+| *External Workspace Manager plugin for Jenkins Pipeline*
+| _Meet the External Workspace Manager plugin, which supports managing workspaces across multiple Jenkins jobs running on different nodes and more!_
+| link:https://github.com/alexsomai[Alex Somai]
+
+| 13:00 - 13:15
+| *Ownership plugin for Jenkins*
+| link:https://speakerdeck.com/onenashev/jw2016-ownership-plugin-demo[Slides]
+
+
+NOTE: will add pictures from lunch demo time general ask the experts booth.
```

Comment:
If you have some available, peppering them throughout the post to give a sense of size for the booth would be cool; it was huge!